packages/executor/src/execution/__tests__/abort-signal.test.ts

```diff
@@ -442,6 +455,89 @@ describe('Abort Signal', () => {
     await expect(next$).rejects.toThrow('This operation was aborted');
     expect(bResolverGotInvoked).toBe(false);
   });
+  it('stops pending stream execution for never-returning incremental delivery (@defer)', async () => {
+    const aResolverGotInvokedD = createDeferred();
```

Comment:
```suggestion
    const aResolverGotInvokedD = createDeferred<void>();
```

Reply:
yeah, started complaining only when I rebased on main

Reply:
done in 1ce4b1ef